Task1/Program.cs

```diff
@@ -27,5 +31,131 @@ public static decimal PerformCalculation(int r)
 
             return square;
         }
+
+        public static void UnpackingArray()
+        {
+            int[] array = { 0, 2, 3, 5, 6 };
+            string[] resArray = new string[array.Length];
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                Console.WriteLine(array[i]);
+            }
+            for (int i = 0; i < array.Length; i++)
+            {
+                for (int j = 0; j < array[i]; j++)
+                {
+                    string nuls = string.Empty;
+                    string ones = string.Empty;
+
+                    if (array[0] == 0 && i == 0)
+                    {
+                        continue;
+                    }
+                    {
+                        if (i % 2 == 0)
+                        {
+                            nuls += "0";
+                            Console.Write(nuls);
+                            continue;
+                        }
+
+                        if (i % 2 != 0)
+                        {
+                            ones += "1";
+                            Console.Write(ones);
+                            continue;
+                        }
+                    }
+                }
+            }
+        }
+
+        public static void Goods()
+        {
+            string[] goods = { "pears", "apples", "cucumbers", "tomatoes", "dill", "parsley", "chicken", "cheese", "butter", "milk" };
+            decimal[] prices = { 40.50M, 31.20M, 21.80M, 35M, 150M, 145M, 200M, 400M, 81.40M, 53.35M };
+            string[] numbers = { "1", "2", "3", "4", "5", "6", "7", "8", "9", "10" };
+            decimal sum = 0M;
```

Comment:
There is a constant value for zero in decimal class.
Please use it instead of zeros everywhere for decimals:
decimal.Zero